supreme_court_predictions/models/logistic_regression.py

```diff
@@ -55,16 +52,14 @@ def __init__(
         self.test_size = test_size
 
         for df in dfs:
-            # Ensure the file exists
-            if os.path.isfile(self.local_path + df):
-                # Use the correct file reading function
-                read_func = (
-                    pd.read_pickle if df.split(".")[-1] == "p" else pd.read_csv
-                )
-                self.dataframes.append(read_func(self.local_path + df))
-
-                # Add name of file
-                self.dataframe_names.append(df.split(".")[0])
+            # Use the correct file reading function
+            read_func = (
+                pd.read_pickle if df.split(".")[-1] == "p" else pd.read_csv
+            )
+            self.dataframes.append(read_func(self.local_path + df))
+
+            # Add name of file
+            self.dataframe_names.append(df.split(".")[0])
```

Comment:
Throughout the rest of the application, it is presume files are there. We should let the models error like we do the rest of the app.